### DISCUSSION TITLE
Install error: Your requirements could not be resolved to an installable set of packages.

### DISCUSSION THREAD

**Discussion by codesinging** [orchestral/testbench#299] [Q&A] [answered]
- Testbench Version: 6.13.0
- Laravel Version: 8.5.12
- PHP Version: 7.4.12
- Database Driver & Version:

### Description:

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - spatie/laravel-ray[1.9.3, ..., 1.14.0] require zbateson/mail-mime-parser ^1.3.1 -> found zbateson/mail-mime-parser[dev-hhvm-tests, dev-travis-test, dev-fix-quoted-printable, dev-master, dev-php-7.4, 0.1.0, ..., 0.4.x-dev, 1.0-alpha, ..., 1.3.0] but it does not match the constraint.
    - orchestra/testbench v6.13.0 requires spatie/laravel-ray ^1.9.3 -> satisfiable by spatie/laravel-ray[1.9.3, ..., 1.14.0].
    - Root composer.json requires orchestra/testbench ^6.13 -> satisfiable by orchestra/testbench[v6.13.0].

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by crynobone** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/172966/110487900-f9874980-8128-11eb-8cd4-2d844a94db14.png)

* `zbateson/mail-mime-parser` version 1.3.1 exists. <https://github.com/zbateson/mail-mime-parser/releases/tag/1.3.1>
* You need to debug why composer wouldn't allow you to install the dependencies. This isn't a bug with Testbench.

**Reply by codesinging** [reply]
Thanks. I found the reason: I use the aliyun mirror(https://mirrors.aliyun.com/composer). The composer mirror works.
